config/config-example.js

```diff
@@ -379,3 +379,10 @@ exports.grouplist = [
 		ip: 's',
 	},
 ];
+
+// punishgroups - the symbols used to mark the different punishment groups, locked and muted
```

Comment:
It would allow server owners to choose what symbol they would want.  I think ``punishments.js`` keeps tracks of all the punishments, but never really deals with the display manifestation of the punish ranks